tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/PoPTests.NetFwk.cs

```diff
@@ -274,6 +275,43 @@ public async Task PopTestWithRSAAsync()
                 result);
         }
 
+        [RunOn(TargetFrameworks.NetCore)]
+        public async Task ROPC_PopTestWithRSAAsync()
+        {
+            var telemetryClient = new TestTelemetryClient(TestConstants.ClientId);
+            var settings = ConfidentialAppSettings.GetSettings(Cloud.Public);
+            var labResponse = await LabUserHelper.GetDefaultUserAsync().ConfigureAwait(false);
+
+            var confidentialApp = ConfidentialClientApplicationBuilder
+                .Create(settings.ClientId)
+                .WithExperimentalFeatures()
+                .WithAuthority(settings.Authority)
+                .WithClientSecret(settings.GetSecret())
+                .WithTelemetryClient(telemetryClient)
```

Comment:
This was marked obsolete recently. Please exclude using it. Can this be validated using the telemetry header for server side telemetry?